backend/alembic/versions/2023_01_05_1144-d4161e384f83_added_messagetreestate_table.py

```diff
@@ -0,0 +1,46 @@
+"""added MessageTreeState table
+
+Revision ID: d4161e384f83
+Revises: 8d269bc4fdbd
+Create Date: 2023-01-05 11:44:02.630633
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "d4161e384f83"
+down_revision = "8d269bc4fdbd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "message_tree_state",
+        sa.Column("id", postgresql.UUID(as_uuid=True), server_default=sa.text("gen_random_uuid()"), nullable=False),
+        sa.Column("created_date", sa.DateTime(), server_default=sa.text("CURRENT_TIMESTAMP"), nullable=False),
```

Comment:
created_date is also already part of message .. so would be redundant. I think we should remove/not add  it here.

Reply:
removed